### ISSUE TITLE
view-angle & fit button

### ISSUE THREAD

**Issue by DaStivi** [smartavionics/RawMouse#10]
hi,
would it be possible to add the predefined view-angle and fit view button?

**Comment by smartavionics** [comment]
Hello. I'm not sure what you mean can you explain further? What mouse device are you using? Does it have more than 2 buttons?

**Comment by DaStivi** [comment]
hi,
yes sorry thought all 3d mouses have them... there are many many buttons! basically i see in the config file the button mapping... but is there any debug output possible to find out what, what button is?

**Comment by smartavionics** [comment]
Yes, set the verbose option in the config file to 1 and it will write messages to the cura.log file when you move the mouse or press a button.

**Comment by DaStivi** [comment]
ok, i see the button numbers... but i'm not able to add them to the config file? i've tried to add for example follwing to lines:
	"7": { "value": 1,	"target": "topview" },
	"15": { "value": 1,	"target": "resetview" }

under the spacemouse and also unter the libspnav "profile" section.. but in both sections it won't work as expected.. :(

btw, i've got the spacepilot mouse with 6 number keys, and al lot of other 3d-view buttons for top,down, left and right view etc..

**Comment by smartavionics** [comment]
Hi, it doesn't recognize "topview", instead, try:

        "7": { "value": [ "y", "90" ], "target": "resetview" },

Left view would require ["x", "90"] and right view ["x", "-90"]

I can't remember if I ever actually tried those, let me know if there is a problem and I will fix.

Don't bother with altering the libspnav profile as that is only used on Linux systems that have that legacy software.
